src/rad/resources/schemas/l1_detector_guidewindow-1.0.0.yaml

```
@@ -0,0 +1,1408 @@
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      signal_resultants:
+        title: Signal Resultants (DN)
+        description: |
+          3D array containing every signal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      lower_left_corners:
+        title: Lower Left Corners
+        description: |
+          2D array containing all the lower left corners of each guide
+          window resultant pair in science coordinates (x_start, y_start).
+        ndim: 2
+        datatype: uint16
+        exact_datatype: true
+      upper_right_corners:
+        title: Upper Right Corners
+        description: |
+          2D array containing all the upper right corners of each guide
+          window resultant pair in science coordinates (x_stop, y_stop).
+        ndim: 2
+        datatype: uint16
+        exact_datatype: true
+      pixel_offsets:
+        title: Pixel Offsets (DN)
+        description: |
+          1D array of constants added to all guide window pixel values in
+          the resultants or a single integer if all values are equal.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            unit: "DN"
+            datatype: uint16
+            exact_datatype: true
+            ndim: 1
+      reset_impacted_pairs:
+        title: Reset Impacted Pairs
+        description: |
+          1D boolean array indicating which pedestal and signal pairs were
+          impacted by reset reads. As reset reads execute, the reset and
+          read individual pixels in a linear fashion across the full detector.
+          Because full frame rows are readout between the pedestal and signal
+          resultants, during a reset read some guide window resultants are
+          impacted by the reset sweeping across those rows of the detector.
+          This flag highlights which guide window resultant pairs are
+          impacted by these resets.
+        ndim: 1
+        datatype: bool8
+        exact_datatype: true
+      reset_read_flag:
+        title: Reset Read Flag
+        description: |
+          1D array of booleans indicating which guide window resultant pairs
+          are read out during a reset. True values correspond to pairs taken
+          during reset reads, False values correspond to pairs taken during
+          normal frame reads.
+        ndim: 1
+        datatype: bool8
+        exact_datatype: true
+      pedestal_times:
+        title: Pedestal Times (UTC)
+        description: |
+          1D array of astropy time objects containing the latched
+          MET times in the UTC standard associated with the pedestal resultants.
+        type: array
+        items:
+          tag: tag:stsci.edu:asdf/time/time-1.*
```

```suggestion
        type: array
        items:
          tag: tag:stsci.edu:asdf/time/time-1.*
```
Will this be a list of `Time` objects or a `Time` object with an array?
I believe the schema will validate:
```
[Time(1, format='jd'), Time(2, format='jd')]
```
but not:
```
Time([1, 2], format='jd')
```

---

@wcschultz The example files have a list of times. Will these be lists of times or a time object with an array?

---

The current plan is a list of times. If there is sufficient reason to switch to an array of times, we can do that instead.